modules/op-setting-resync-period.adoc

```diff
@@ -0,0 +1,38 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-setting-resync-period_{context}"]
+= Setting the resync period for the pipelines controller
+
+You can configure the resync period for the pipelines controller. Once every resync period, the controller reconciles all pipeline runs and task runs, regardless of events. 
+
+The default resync period is 10 hours. If you have a large number of pipeline runs and task runs, a full reconciliation every 10 hours might consume too many resources. In this case, you can configure a longer resync period.
+
+.Prerequisites
+* You are logged in to your {OCP} cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. In the `TektonConfig` CR, configure the resync period for the pipelines controller, as shown in the following example.
++
+.Example of setting the resync period for the pipelines controller to 24 hours
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    options:
+      deployments:
+        tekton-pipelines-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: tekton-pipelines-controller
+                  args:
+                    - "-resync-period=24h"
```

Comment:
Suggestion - since you don't mention which specific value to modify in the actual step, would it be helpful for users if you added a callout label here and explained the variable underneath the example?
```suggestion
                    - "-resync-period=24h" <1>
```